what4/src/What4/Protocol/Online.hs

```diff
@@ -236,6 +239,29 @@ checkSatisfiable proc rsn p =
         do assume conn p
            check proc rsn
 
+-- | Get `n` abducts from the SMT solver, the disjunction of which entail `t`, and bind them to `nm`
```

Comment:
What does it mean to "bind" the abducts? My impression is that getting the first abduct binds it to a name and then you have to use the same name in subsequent queries?

Could you make that description a bit more explicit if that is the case?

Does that name persist after getting `n` abducts? Is it scoped to the current frame?